### DISCUSSION TITLE
USB boot and shutdown

### DISCUSSION THREAD

**Discussion by aerialshooter** [gopro/labs#670] [Q&A]
Can anyone help me with the correct code to make a Hero 7 (Black) boot up when USB power is applied and shutdown when USB power is removed?

Thanks

**Comment by dnewman-gpsw** [comment]
That is not possible on HERO7, only later models.

**Reply by aerialshooter** [reply]
Thanks

**Comment by msdogo** [comment]
hero12 how to do？

**Reply by dnewman-gpsw** [reply]
Can expand your question, what have you tried, what are you trying to do?  There are several threads on setting this up for for hero 8 through 12.  Look up !MWAKE=2 and conditional scripting.  Also many dashcam posts that discuss this. Auto booting the camera is easy with the command above, but what you do for shutting down the camera is very application specific.

**Reply by msdogo** [reply]
thank you,  This instruction achieves my needs！
